pkg/utils/cluster.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"context"
+
+	"github.com/golang/glog"
+
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func IsSingleNodeCluster(c client.Client) (bool, error) {
+	nodeList := &corev1.NodeList{}
+	err := c.List(context.TODO(), nodeList)
+	if err != nil {
+		glog.Errorf("Failed to list nodes: %v", err)
+		return false, err
+	}
+
+	if len(nodeList.Items) == 1 {
+		glog.Infof("only one node found in the cluster marking disableDrain as true")
```

Comment:
nit: i dont think this log is needed.

if you want to keep it, please update the message as disableDrain is now out of context.

Reply:
I will like to leave the message makes debug easier :)

I just change the comment